src/stories/AddMovie.stories.js

```diff
@@ -0,0 +1,19 @@
+import { MovieForm } from "../entities/MovieForm/MovieForm";
+import { Dialog } from "../shared/ui";
+
+export default{
```

Comment:
why do you need 3 files for one component? let's create one story file with default variant and 2 other options with different params

Reply:
Sorry, I didn't quite understand this.( Could you please clarify what I should do?

Reply:
You need to create  a template for the component `Dialog`, kind of default state, and create several states with different props. Now you do the same thing but you do it in 3 files for some reasons instead of one. 

something like that

```suggestion
Default.args = {
    title: "Add Movie",
    children: <MovieForm/>
  };
  
  Delete.args = {
    title: "Delete Movie",
    children: (
    <div>
        <p style={{marginBottom: '30px'}}>Are you sure you want to delete this movie?</p>
        <button style={{padding: '4px 20px', cursor: 'pointer'}}>Confirm</button>
    </div>
  ),
  };
  
  Edit.args = {
  title: "Edit Movie",
  children: (
    <MovieForm
      initialMovie={{
        title: "Pulp fiction",
        movieUrl: "https://www.imdb.com/title/tt0110912/?ref_=tt_mv_close",
        genre: "CRIME",
        overview: "Decription",
        releaseDate: '1994-05-06',
        rating: 8.9,
        runtime: "2h 10min"
      }}
    />
  ),
};
  

```

Reply:
okaay I've got it , thanks